caesars-palace/blocks/intro-section/intro-section.js

```diff
@@ -0,0 +1,21 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
+  const rows = [...block.children];
+  // section inner
+  const innerContainer = rows[0];
+  innerContainer.classList.add('intro-section-inner');
+  // content
+  const mainChildren = [...innerContainer.children];
+  mainChildren[0].classList.add('intro-section-content');
+  // main image
+  const mainImageContainer = mainChildren[1];
+  mainImageContainer.classList.add('intro-section-image-container');
+  // secondary image.  move to section inner.
+  const secondImageContainer = rows[1];
+  const secondImageDiv = secondImageContainer.firstElementChild;
+  secondImageDiv.classList.add('intro-section-secondary-image-container');
+  innerContainer.append(secondImageDiv);
+  secondImageContainer.remove();
+  block.querySelectorAll('img').forEach((img) => img.closest('picture').replaceWith(createOptimizedPicture(img.src, img.alt, false, [{ width: '750' }])));
```

Comment:
Should `width: '750'` reflect the breakpoints defined on th Caesars site? I'm still a little unclear on how to best use this function effectively.

Reply:
That's a great question.  I'd like some guidance on this as well.  I was going to set it to the largest width that CSS would allow for the largest viewport (i.e. `872px` for the main image).  That still may make sense.

@amol-anand - Thoughts?

Reply:
I went ahead and removed this call as @amol-anand had mentioned in Slack that these images are already optimized.